src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="sky-vertical-tabset-group">
   <div
+    *skyThemeIf="'default'"
```

Comment:
I'm curious in why we are adding this `skyThemeIf` here and then changing how the `skyThemeIf`s on the chevron and modern chevron equivalent are working? Unless I'm missing something we are now achieving the same thing with duplicated code of this `div` down below.

Reply:
@Blackbaud-TrevorBurch It seemed like I needed to add the `skyThemeif` at that level because the ngclass on that div needed to be different btw modern and default. This wasn't done on the initial styles updates. The modern version uses a different font class (sky-font-emphasized vs font-emphasized) to achieve a bolder font weight in modern style. Maybe there is a better way to do this? I just did it at the top level div b/c there was this change the chevrons were different.

Reply:
Removed the duplicated div and added a modern class (sky-font-emphasized) to the subMenuOpen state to achieve the modern font weight instead.